### DISCUSSION TITLE
Prisma cannot serialize bigInt

### DISCUSSION THREAD

**Discussion by k11q** [nuxt/nuxt#16366] [Questions]
Hi I am using prisma to generate my api, database from supabase. I get 500 error when trying to fetch the api and also when go to the route, it states cannot serialize bigInt, which I use for my ids in my tables. What is the solution to this?

**Comment by Zeglius** [comment]
Do this to specify the kind of interger as for
 https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#bigint

```
model Whatever{
    Id       Int        @db.BigInt
}
